homeassistant/components/purpleair/sensor.py

```diff
@@ -175,7 +174,7 @@ class PurpleAirSensorEntityDescription(
         key="voc",
         name="VOC",
         device_class=SensorDeviceClass.VOLATILE_ORGANIC_COMPOUNDS,
-        native_unit_of_measurement=CONCENTRATION_IAQ,
+        native_unit_of_measurement=CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
```

Comment:
Is this the unit that the measurement is measured in?

Reply:
I thought so, but after reviewing [the API docs](https://api.purpleair.com/), what I had before ("Bosch static iaq units as per BME680 spec sheet") is the correct unit. I think the right thing to do here is just remove the device class.